packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -351,6 +352,32 @@ export async function getBookings({
       take: take + 1,
       skip,
     }),
+    prisma.booking.findMany({
+      where: {
+        OR: [
+          {
+            eventType: {
+              parent: {
+                team: {
+                  members: {
+                    some: {
+                      userId: user.id,
+                      role: {
+                        in: ["ADMIN", "OWNER"],
+                      },
+                    },
+                  },
+                },
+              },
+            },
```

Comment:
For a managed child event we have a parent event which is connected to a team, and we just make sure that the user querying is an Admin of such a team.